src/models/backingImage.js

```diff
@@ -74,6 +74,25 @@ export default {
       })
       yield put({ type: 'query' })
     },
+    *startWS({
+      payload,
+    }, { select }) {
+      let ws = yield select(state => state.backingImage.ws)
```

Comment:
Should it be selected from the state directly w/o namespace? like `state.ws`?

Reply:
this state‘s struct like `{
  app: { state : },
  volume:{ state : }, ,
  settings: { state : },,
  eventlog...,
  engineimage...,
  backingImage..,
}`
So need state.backingImage.ws